### DISCUSSION TITLE
create several instances

### DISCUSSION THREAD

**Discussion by SoenkeHob** [Hofknecht/SystemTrayMenu#374] [Q&A] [answered]
I love SystemTrayMenu. On Windows 10 I used folder shortcuts in the taskbar but for whatever reason Microsoft got rid of that functionality. Lucky me, I found SystemTrayMenu. 
It would be great though if I could have several shortcuts from the taskbar directly. E.G one for client folders, another one for regulations, one for documents... I know I can have those three folders as start folders but I would like to skip that step and be able to open directly from the taskbar the three folders.

**Comment by topeterk** [comment, marked as answer]
For about 4 months this came up already but was rejected. See: #252
As you can see in the marked issue that I also suggested this pretty early but maybe the decision can be reconsidered.
However implementing this would change a bunch of things that would become much more difficult which I think is still the case.

**Reply by SoenkeHob** [reply]
Great, thank you for the feedback. It was just a suggestion as I used to have in Windows 10 various explorer shortcuts in my taskbar but I am getting more and more used to the tool you have developed, which is a great help. 
I got another suggestion. Not sure if that should be a new conversation:
It would be great if one could drag and drop or drag a copy and drop out of SystemTrayMenu

**Reply by Tanjalibertatis** [reply]
> I got another suggestion. Not sure if that should be a new conversation: It would be great if one could drag and drop or drag a copy and drop out of SystemTrayMenu

It is possible to copy an item via drag and drop e.g. to the desktop, or to drag and drop something into an application (#34) in order to open it.
Do you mean that? Thanks for your feedback!

**Comment by SoenkeHob** [comment]
But one cant drop it into a different folder in Windows Explorer. At least in most cases where I tried. It works only sometimes.

**Comment by Hofknecht** [comment]
@SoenkeHob 
Can you try whether it might always work if no scrollbar is displayed. Then maybe it's the swipe scroll
https://github.com/Hofknecht/SystemTrayMenu/issues/275
We could make a setting for this so that it is only enabled by default for touch devices or a setting to disable it.
If the scrollbar is displayed, you have to pull the item out directly without hovering over another item in order for it to work, otherwise we will interpret it as a swipe scroll and stop the drag drop

**Comment by SoenkeHob** [comment]
Hallo Markus
 
Richtig, das scheint es zu sein. 
Aus diesem Ordner kann ich kopieren

 
Aber aus diesem kann ich es nicht:

 
Lieben Gruß
Sönke

**Comment by Hofknecht** [comment]
@SoenkeHob 
implemented options for drag, so it can be configured now
https://github.com/Hofknecht/SystemTrayMenu/issues/377
now swipe scolling is default only enabled at touch devices and should not more disturb
if you have the chance try to compile latest version and try it out [1.2.9.5](https://github.com/Hofknecht/SystemTrayMenu/commit/587ebcfc90c30d572f4dec6c3deef8249b8f65c2)

don't hestiate to write if you have more questions

**Comment by SoenkeHob** [comment]
Thanks Markus
 
How do I do that?
 
Cheers
Sönke

**Comment by Hofknecht** [comment]
@SoenkeHob 
https://github.com/Hofknecht/SystemTrayMenu#build
Install Visual Studio 2022 (community version, free): https://visualstudio.microsoft.com/de/thank-you-downloading-visual-studio/?sku=Community&channel=Release&version=VS2022&source=VSLandingPage&cid=2030&passive=false
get the code: https://github.com/Hofknecht/SystemTrayMenu/archive/refs/heads/master.zip
double click SystemTrayMenu.sln and in visual studio press F5

other option is to wait until we make a new release and test it there

**Comment by Hofknecht** [comment]
d.schuermann voted for that topic. we will have a look after released 1.3

**Comment by SoenkeHob** [comment]
When will it be released?
I was not able to work via Visual Studio. Am not advanced enough. 
Would be nice to have the drag and drop fixed. 
 
Greetings
Sönke

**Comment by Hofknecht** [comment]
@SoenkeHob 
Today we released 1.3.0.0   : )
Find it here:
https://github.com/Hofknecht/SystemTrayMenu/releases

**Comment by Hofknecht** [comment]
B B via mail asked/ voted for that Topic
we will review again how we can realize that topic

**Comment by SoenkeHob** [comment]
Hello Markus
 
Just to let you know, the systemtraymenu has updated itself some days back and is working beautifully. 
I can copy and/or drag&drop documents out of any folder into WinExplorer or Outlook without any hassles. 
 
Thank you again for this great tool. 
I use it a couple of times per hour. In fact, I open most folders and documents via this tool. 
 
All the best
Sönke

**Comment by SoenkeHob** [comment]
Thanks you Markus
 
How do I see, if the previous version is installed or portable?
And if either, do I need to deinstall it first before loading the new version?
If I already have the installed version, should I then update or re-install via the Microsoft Store as recommended on the website?
 
Thank you for you help
Sönke

**Reply by Hofknecht** [reply]
@SoenkeHob 
your question is attached to #374 but not related to it, i will hide it in the discussion here
https://github.com/Hofknecht/SystemTrayMenu/discussions/374

answer to your question:

for all versions:
-you can start the about menu via context menu, there you will see the version in format like 1.3.1.0
-you can check for latest version via context menu -> Check for updates (this will not install, only inform you)

for portable version:
-you dont see other versions, you can have as many versions as you want.
-you can delete it via delete the folder, you not need to, but i would recommend to just replace the old
(e.g. for autostart, the last instance wins where you pressed settings -> OK)

for Microsoft store version:
-you can only have one version the latest
-you can delete via store, you not need to deinstall before, it will update automatically (maybe to trigger it you need to open microsoft store, or maybe even open the page in microsoft store via search 'systemtraymenu')

hope that helps, if there are more let us know, maybe write me directly or open new discussion

**Comment by Hofknecht** [comment]
just to clarify, we do still not support several instances, it is only possible to have one main menu

would be nice to have several taskbar items like tortoise:
![show instances like gittortoise](https://user-images.githubusercontent.com/52528841/196981449-8234b287-f50e-4fe9-a688-86c4f669e5c1.png)

but need to think more about it how to realize that topic

**Comment by SoenkeHob** [comment]
Thanks Markus
 
At some stage all over the sudden there was a new version installed without me doing anything:

 
It looks a bit different and all works perfectly fine now incl. drag and drop. I particularly enjoy that it doesn’t close now when dragging out of it so one can drag several documents after each other or even open another folder after dragging out. 
 

 
Nice work! Thank you very much again. 
 
Greetings
Sönke
